### DISCUSSION TITLE
"javascript_importmap_shim_nonce_configuration_tag" occurred error in Rails7.1.2 with importmap

### DISCUSSION THREAD

**Discussion by ecmerchant** [railsadminteam/rails_admin#3666] [General]
Since "importmap_shim" is dropped at Rails7.1.2 & importmap, "javascript_importmap_shim_nonce_configuration_tag" in _head.html.erb occurred error.
I hope someone fix it. Thanks.

**Comment by mshibuya** [comment]
Fixed by https://github.com/railsadminteam/rails_admin/commit/24380d53f7c82eef4758e96a0b98e9d4333a806d, thanks.

**Comment by Halphas** [comment]
Hello,

I updated to rails 7.1.3 but my rails_admin is now throwing an error.  
I tried to update rails_admin with bundle update, but I guess the release with this fix is not out yet. Can you tell me when it will be released? I really don't want to touch my auto build source files and include the master branch or anything Thanks!
